### DISCUSSION TITLE
Struct property not available

### DISCUSSION THREAD

**Discussion by developer2346** [NomicFoundation/hardhat#4077] [Troubleshooting] [answered]
This is my Solidity code:
<code>struct Person {
        bool exists;
        Card[8] cards;
    }
mapping(address => Person) public Persons;
</code>
This is my TS code:
<code>const prsns = await contract.Persons(address)
// prsns.cards does not exist</code>

How can I access to Person.cards? It is not available

**Comment by clauBv23** [comment, marked as answer]
I assume that you are calling the default getter generated by the compile, in that case, the default getter won't return arrays in order to reduce gas costs. You need to define a getter and use it, then you'll be able to get `person.card`
